benchmark-service/benchmark/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.7-slim-stretch
+
+COPY benchmark-service/benchmark /benchmark-service/benchmark/
+
+COPY benchmark-service/setup.py benchmark-service/MANIFEST.in  /benchmark-service/
+
+RUN pip install aiohttp aiohttp_jinja2
```

Comment:
Move this line `RUN pip install aiohttp ...`  to the first step after `FROM python:3.7-slim-stretch`. The reason for it is these packages are not going to change, but you might change the scripts many times. You can cache layers that have no changes to them and your Docker image will build faster each time. Let me know if that doesn't make sense.

Also, use `pip3` instead of `pip`